src/app/_locales/en/quick-start-guide-ios.md

```diff
@@ -0,0 +1,21 @@
+### Welcome to this project, you will need two main steps to make the extension work:
+
+- **Open the App and set a directory**
+
+  - This directory is used to saving and loading your userscripts.
+  - A default local directory is already set when you first launch the App.
+  - Click the current directory to jump to the default directory or the one you set.
```

Comment:
```suggestion
- **Open the app and set a directory**

  - When you first launch the app, a local directory will be set by default.
  - If needed, you can select a new directory to suit your preferences.
  - This directory is used for saving and loading your user scripts.
  - Click the current directory to jump to the default directory or the one you set.
```

I think this conveys the message more clearly. Let me know if I missed the point of this section.